# Fix: src/boxed.hh:39:30: error: no type named 'in_place_t' in namespace 'std'

--- a/src/boxed.hh
+++ b/src/boxed.hh
@@ -19,6 +19,7 @@
 // but we need this for non-enum/integral/floating types.
 
 #include <type_traits>
+#include <utility>
 
 namespace vte {
 
